tests/dummy/app/initializers/router-ext.js

```diff
@@ -10,12 +12,28 @@ function initialize() {
   Ember.Router.reopen({
     assetLoader: Ember.inject.service(),
 
-    _getHandlerFunction() {
-      const originalFunction = this._super(...arguments);
+    setupRouter() {
+      let isSetup = this._super(...arguments);
+      if (newSetup) {
```

Comment:
I don't fully understand the point of this conditional, it is never set to any other value than `true`. Can you remove?

Reply:
done!